package.json

```diff
@@ -9,5 +9,9 @@
         "axios": "^1.1.2",
         "laravel-vite-plugin": "^0.8.0",
         "vite": "^4.0.0"
+    },
+    "dependencies": {
+        "@prettier/plugin-php": "^0.20.1",
+        "prettier": "^3.0.3"
```

Comment:
this should not add to the project dependency.

you should have install them **Globally** on your system & not in the project.

remove them from here please.